.github/workflows/analyse.yml

```diff
@@ -0,0 +1,28 @@
+name: PHPStan
+
+on: ['push', 'pull_request']
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    name: analyse
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.2
+          extensions: dom, curl, libxml, mbstring, zip, pdo, sqlite, pdo_sqlite, bcmath, soap, intl, gd, exif, iconv, imagick, fileinfo
+          coverage: none
+
+      - name: Repository Credentials
+        run: composer config repositories.foomanmirror composer https://repo-magento-mirror.fooman.co.nz/
```

Comment:
I don't think we need this anymore since the repository is already configured in the composer.json
```suggestion
```